app/components/grouped_cards/listing_component.rb

```diff
@@ -15,7 +15,7 @@ def group_link_anchor(group)
     end
 
     def items(group)
-      @data.dig(group, "providers")
+      @data.dig(group, "providers").sort_by { |provider| provider["header"].to_s.downcase }
```

Comment:
I think it would be more efficient to do the sort _once_ when we generate the Yaml data (see `lib/tasks/generate_assessment_only_providers.rake`), rather than sort here which will run on _every request_ of the page.